code/drasil-printers/lib/Language/Drasil/Debug/Print.hs

```diff
@@ -239,38 +235,7 @@ mkTableReferencedChunks PI { _ckdb = db } =
     testIsolateLayout (x, ys) = text (show x) $$ nest nestNum (text $ show ys)
 
     refbyUIDs :: [(UID, [UID])]
-    refbyUIDs = Map.assocs $ db ^. refbyTable
-
-    nestNum = 30
-
--- | Chunks that use and are used by other chunks.
-mkTableDepReffedChunks :: PrintingInformation -> Doc
-mkTableDepReffedChunks PI { _ckdb = db } = text
```

Comment:
Not a very helpful chunk dump table because of readability issues. So, rather than fixing it, I opted to remove it. The same functionality will be rebuilt soon anyways.